### DISCUSSION TITLE
Extreme visit duration for certain days

### DISCUSSION THREAD

**Discussion by joseika** [plausible/analytics#580] [Anything Else]
I've tried out Plausible for a site i develop for 2 weeks. For last week I get extreme visit durations in the dashboard for 2 days, January 8 & 10. All the other days look normal, around 3 minutes.
![image](https://user-images.githubusercontent.com/33547290/104158586-f3594300-53ed-11eb-8d4a-9990ce7d65c1.png)

Anyone else seeing something similar, or ideas to why/how this happens?

**Comment by metmarkosaric** [comment]
Looks weird @joseika! (you also have a very very low bounce rate) And thanks for reporting this! Would you be able to reach out to us with more details so we can take a look at your site and see what we can find out? https://plausible.io/contact . Obviously you can also post here if you don't mind sharing your site URL. Thanks

**Reply by joseika** [reply]
Hi, @metmarkosaric. I'm drafting an email with some other challenges we've seen with our site/needs, and regarding the API on your roadmap. I'll add the details there. You should have it around lunchtime today.

**Reply by metmarkosaric** [reply]
Sounds good, thanks!

**Comment by ukutaht** [comment]
I have identified the issue and we are testing a fix for this: https://github.com/plausible/analytics/pull/593

Fortunately this problem only affected a handful of session on your account @joseika  so I can restore your dashboard once the fix is tested and live.
